release-notes.md

```diff
@@ -1,19 +1,49 @@
+2.0.0-RC7
+---------
+* CI/CD implementation
+* Delivery Tracker - limit requests map per peer
+* add request tracker actor
+* add upper bound limit for peers database
+* rename Scorex -> Sparkz
+* update lib versions to latest, check and fix vulnerabilities
+* KnownPeers should never be discarded
+* exclude unconfirmedConnections and already active connections
+* DnsClient lookup flow implementation
```

Comment:
Are we sure this is will be included here? I didn't find it in this branch and I think it requires an additional round of review

Reply:
correct, i will remove